CommandPattern.java

```diff
@@ -0,0 +1,132 @@
+package Ex4;
+import java.util.Scanner;
+public class CommandPattern {
+	public static void main(String[] args) {
+		UniversalRemote universalRemote=new UniversalRemote();
+		
+		Tv tv=new Tv();
+		SetTopBox sbox=new SetTopBox();
+		SoundSystem ss=new SoundSystem();
+		VGame vgame=new VGame();
+		
+		Command newscommand=new NewsChannelCommand(tv, sbox, ss, vgame);
+		Command serialcommand=new SerialChannelCommand(tv, sbox, ss, vgame);
+		Command ttgamecommand=new TTGameCommand(tv, sbox, ss, vgame);
+		
+		universalRemote.assignCommand(1, newscommand);
+		universalRemote.assignCommand(2, serialcommand);
+        universalRemote.assignCommand(3, ttgamecommand);
+        
+        Scanner scan=new Scanner(System.in);
+        System.out.println("Enter the Command that need to be executed..");
+        int input=scan.nextInt();
+		universalRemote.executeCommand(input);
+	}
+}
+class UniversalRemote{
+	Command comm[]=new Command[5];
```

Comment:
Why static value 5 is used?

Reply:
Just for this use case I have kept it as Static value bro.